test/bootstrap_fields_test.rb

```diff
@@ -245,11 +257,48 @@ class BootstrapFieldsTest < ActionView::TestCase
       </form>
     HTML
     assert_equivalent_xml expected, output
+
+    ## TODO: DRY up the tests for the _with version
+    if Gem::Version.new(::Rails::VERSION::STRING).release >= Gem::Version.new('5.1.0')
+      output = bootstrap_form_with(model: @user, layout: :horizontal, label_col: 'col-sm-2', control_col: 'col-sm-10', local: true) do |f|
+        f.fields :address do |af|
+          af.text_field(:street)
+        end
+      end
+      assert_equivalent_xml remove_default_ids_for_rails_5_1(expected.gsub(/ class="new_user" id="new_user"/, "")),
+                            output
+    end
+  end
+
+  # This test demonstrates that the Rails `fields` method passes its options
+  # to the builder, so there is no need to write a `bootstrap_form` helper
+  # for the `fields` method. If this test fails, it probably means you have to
+  # write a `fields` helper in `lib/bootstrap_form/form_builder.rb`.
```

Comment:
It seems this test was written to investigate Rails behavior, and is now no longer needed. Would you agree?

Reply:
I had the same thought, but then I thought it would also serve to test the assumption going forward and give us a warning if something changes in a future Rails version that the assumption is no longer valid. I think there's value to leave it in (perhaps with a better comment to make the intent clearer), but if you want it removed I'll certainly do so.